front/app/modules/commercial/content_builder/admin/components/CraftComponents/TwoColumn/index.tsx

```diff
@@ -22,18 +35,98 @@ const TwoColumn: UserComponent = () => {
       w="100%"
       gap="4px"
     >
-      <Box flex="1">
+      <Box
+        flex={columnLayout === '1-1' ? '1' : columnLayout === '1-2' ? '1' : '2'}
+      >
         <Element id="column1" is={Container} canvas />
       </Box>
-      <Box flex="1">
+      <Box
+        flex={columnLayout === '1-1' ? '1' : columnLayout === '1-2' ? '2' : '1'}
+      >
         <Element id="column2" is={Container} canvas />
       </Box>
     </Box>
   );
 };
 
+const TwoColumnSettings = () => {
+  const {
+    actions: { setProp },
+    columnLayout,
+  } = useNode((node) => ({
+    columnLayout: node.data.props.columnLayout,
+  }));
+
+  return (
+    <Box mt="-30px" mb="30px">
+      <Title variant="h4" as="h3">
+        <FormattedMessage {...messages.columnLayoutRadioLabel} />
+      </Title>
+      <Radio
```

Comment:
Let's check this one by default when you first create the component.

Reply:
@IvaKop This one should already be checked when you first create it. Is it not working for you?

Reply:
No, when I first create it, nothing is selected.

Reply:
Hmm alright I'll take a look. Maybe it's a change in my current branch. Thanks!